source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/edit/form/index.tsx

```diff
@@ -38,18 +46,22 @@ class EditProfileDisplay extends React.Component<IProps & i18nProps, IState> {
 
   submit = async (e) => {
     e.preventDefault();
-    await this.props.onSubmit({ ...this.state });
+    const profileVisibility = this.state.profileVisibility ? PUBLIC_PROFILE : PRIVATE_PROFILE;
+    await this.props.onSubmit({ ...this.state, profileVisibility });
   }
 
   render() {
     const { mut, toggle } = this;
     const {
       givenName, familyName, email, phone,
-      timezone, emailNotification
+      timezone, emailNotification,
+      profileVisibility
     } = this.state;
 
     const { t } = this.props;
 
+    const profileVisibilityText = profileVisibility ? t('profile.visibility.visible') : t('profile.visibility.restricted');
```

Comment:
this line is too long, imo
![image](https://user-images.githubusercontent.com/199018/44274936-7d972000-a211-11e8-807e-254bf27a90e8.png)

we should probably find a lint rule to restrict line length.